src/TDetachedWindow.cpp

```diff
@@ -957,31 +990,40 @@ void TDetachedWindow::updateWindowTitle()
 
         if (!mCurrentProfileName.isEmpty()) {
             pHost = mudlet::self()->getHostManager().getHost(mCurrentProfileName);
+            //: This is the title of a Mudlet window which was detached from the main Mudlet window, and %1 is the name of the profile.
             title = tr("Mudlet - %1 (Detached)").arg(mCurrentProfileName);
         } else {
+            //: This is the title of a Mudlet window which was detached from the main Mudlet window, but has no profile loaded.
             title = tr("Mudlet (Detached)");
         }
 
         if (pHost) {
+            title += qsl(" - ");
             bool isConnected = (pHost->mTelnet.getConnectionState() == QAbstractSocket::ConnectedState);
             bool isConnecting = (pHost->mTelnet.getConnectionState() == QAbstractSocket::ConnectingState);
 
             if (isConnected) {
-                title += tr(" - Connected");
-
                 if (!pHost->getUrl().isEmpty()) {
-                    title += tr(" to %1").arg(pHost->getUrl());
+                    //: This text will be added to the title of a detached Mudlet window, if it is currently connected. The whole title will be like "Mudlet PROFILENAME (Detached) - Connected to GAMENAME"
+                    title += tr("Connected to %1").arg(pHost->getUrl());
+                } else {
+                    //: This text will be part of to the title of a detached Mudlet window, if it is currently connected but we don't know to where. The whole title will be like "Mudlet PROFILENAME (Detached) - Connected"
+                    title += tr("Connected");                    
                 }
             } else if (isConnecting) {
-                title += tr(" - Connecting...");
+                //: This text will be part of the title of a detached Mudlet window, if it is about to be connected. The whole title will be like "Mudlet PROFILENAME (Detached) - Connecting..."
+                title += tr("Connecting...");
             } else {
-                title += tr(" - Disconnected");
+                //: This text will be part of the title of a detached Mudlet window, if it is not connected. The whole title will be like "Mudlet PROFILENAME (Detached) - Disconnected"
+                title += tr("Disconnected");
             }
         }
     } else {
         // Multiple profiles - show count and current
+        //: This is the title of a Mudlet window which was detached from the main Mudlet window, and has multiple profile opened. %1 is the number of profiles, %2 is the name of the profile currently shown.
```

Comment:
Does "%1 is the number of profiles" mean in this detached window or in total across the whole Mudlet applications? *Just asking!*